tests/cql/CqlTypesTest.xml

```diff
@@ -156,15 +156,15 @@
 	<group name="Time">
 		<test name="TimeUpperBoundHours">
 			<expression invalid="true">@T24:59:59.999</expression>
-			<!-- Invalid date-time input (T24:59:59.999). Use ISO 8601 date time representation (yyyy-MM-ddThh:mm:ss.mmmmZhh:mm). -->
+			<!-- Translation Error: Invalid date-time input (T24:59:59.999). Use ISO 8601 date time representation (yyyy-MM-ddThh:mm:ss.mmmmZhh:mm). -->
```

Comment:
These errors were noted before already, but I thought it was important to note that they are _translation_ errors, not _runtime_ errors.  In my approach, where I am using these tests only to test the runtime, the distinction betwee a translation error and runtime error is important (since translation errors are not a concern of my software).